packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -139,19 +154,11 @@ const start = async (zcf, privateArgs) => {
     )}`,
   );
 
-  const { wrapPublicFacet, wrapCreatorFacet, getNat, getInvitationAmount } =
-    handleParamGovernance(zcf, paramManager);
-
-  const electorateInvAmt = getInvitationAmount(CONTRACT_ELECTORATE);
-  assert(
-    keyEQ(electorateInvAmt, electorateParam.value),
-    X`electorate amount (${electorateParam.value} didn't match ${electorateInvAmt}`,
```

Comment:
I took it out because I thought it was tautological. Maybe I misread the code. It looked like it came into this function, was passed into `handleParamGovernance` and then this check was that what came back out matched what was passed in. ??

Reply:
I think the assert verifies that the invitation amount known to parameter governance (which was initialized based on `privateArgs`) matches the one that was specified in `terms`. Those are required to be the same, but not tautalogically.

Reply:
Got it, thanks. Made re-usable in d45aa114334b161778fb30cc337ed55da97a5308

Reply:
> d45aa114334b161778fb30cc337ed55da97a5308

coming soon? I don't see it in the list of commits in this PR. Is there a different way to look for a commit?

Reply:
Oh, that looks good! Thanks for adding it in all the right places.

Except I'm hoping to [hide paramManager from the contract](https://github.com/Agoric/agoric-sdk/issues/4740).

I think that won't be too bad. When I refactor that, the call to `assertElectorateMatches()` can go inside `handleParamGovernance()`.